sherpa-onnx/csrc/provider-config.cc

```diff
@@ -19,7 +19,7 @@ void CudaConfig::Register(ParseOptions *po) {
 bool CudaConfig::Validate() const {
 
   if(cudnn_conv_algo_search > 0 && cudnn_conv_algo_search < 4) {
-    SHERPA_ONNX_LOGE("cudnn_conv_algo_search: '%d' is not valid option."
+    SHERPA_ONNX_LOGE("cudnn_conv_algo_search: '%d' is not a valid option."
```

Comment:
If `cudnn_conv_algo_search` is 1, this if statement is also executed, right?
1 is a valid value since the log says `[1, 3]`.

Reply:
yes, I will update it